llvm/test/Transforms/WholeProgramDevirt/import.ll

```diff
@@ -9,11 +9,25 @@
 ; RUN: opt -S -passes=wholeprogramdevirt -wholeprogramdevirt-summary-action=import -wholeprogramdevirt-read-summary=%S/Inputs/import-branch-funnel.yaml < %s | FileCheck --check-prefixes=CHECK,BRANCH-FUNNEL,BRANCH-FUNNEL-NOVCP %s
 
 ; Cutoff value is not explicitly set. Expect 3 remark messages.
-; RUN: opt -S -passes=wholeprogramdevirt -wholeprogramdevirt-summary-action=import -pass-remarks=wholeprogramdevirt -wholeprogramdevirt-read-summary=%S/Inputs/import-single-impl.yaml < %s  2>&1 | grep "single-impl" | count 3
+; RUN: opt -S -passes=wholeprogramdevirt \
+; RUN: -wholeprogramdevirt-summary-action=import \
```

Comment:
Nit: I normally find two space indentation for additional parts of command on new lines to be more readable.